dev/io.openliberty.microprofile.telemetry.2.1.internal_fat_tck/publish/tckRunner/tck/pom.xml

```
@@ -0,0 +1,253 @@
+		<arquillian.liberty.managed.jakarta.version>2.1.1</arquillian.liberty.managed.jakarta.version>
+		<testng.version>7.5.1</testng.version>
+		<slf4j.version>2.0.6</slf4j.version>
+
+        <opentelemetry.version>1.35.0</opentelemetry.version>
+        <mptelemetry.version>2.1-RC1</mptelemetry.version>
+
+        <!-- the below is used in arquillian.xml -->
+        <wlp>${wlp}</wlp>
+        <!-- <wlp>${project.basedir}/../../../../build.image/wlp/</wlp> -->
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <defaultSuiteFiles>tck-suite.xml</defaultSuiteFiles>
+        <suiteXmlFile>${defaultSuiteFiles}</suiteXmlFile>
+        <targetDirectory>${project.basedir}/target</targetDirectory>
+        
+    </properties>
+
+    <groupId>io.openliberty.microprofile.telemetry</groupId>
+    <artifactId>tck.runner.tck</artifactId>
+    <version>1.0</version>
+    <name>MicroProfile Telemetry ${mptelemetry.version} TCK Runner TCK Module</name>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>com.ibm.ws</groupId>
+            <artifactId>fattest.simplicity</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/../../../lib/fattest.simplicity.jar</systemPath> <!-- This will point to build/libs/autoFVT/lib/fattest.simplicity.jar -->
+        </dependency>
+        
+        <dependency>
+            <groupId>org.eclipse.microprofile.telemetry</groupId>
+            <artifactId>microprofile-telemetry-tracing-tck</artifactId>
+            <version>${mptelemetry.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.telemetry</groupId>
+            <artifactId>microprofile-telemetry-metrics-tck</artifactId>
+            <version>${mptelemetry.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.telemetry</groupId>
+            <artifactId>microprofile-telemetry-logs-tck</artifactId>
+            <version>${mptelemetry.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.microprofile.rest.client</groupId>
+            <artifactId>microprofile-rest-client-api</artifactId>
+            <version>3.0</version>
+            <scope>system</scope>
+            <systemPath>${io.openliberty.org.eclipse.microprofile.rest.client.3.0_}</systemPath>
```

```suggestion
            <systemPath>${io.openliberty.org.eclipse.microprofile.rest.client.4.0_}</systemPath>
```
For MP 7.1 it isn't 3.0.